core/src/main/java/io/fabric8/maven/core/service/BuildService.java

```diff
@@ -43,6 +43,9 @@
      */
     void postProcess(BuildServiceConfig config);
 
+    default BuildServiceConfig getBuildServiceConfig() {
```

Comment:
Is this needed? I don't actually see any class implementing the method neither invocations.